living_documentation_regime/README.md

```diff
@@ -0,0 +1,358 @@
+
+### Hosting Documentation Tickets in a Solo Repository
+
+Using a dedicated repository solely for documentation tickets provides multiple advantages:
+- **Streamlined Management:** This avoids cross-project conflicts and board exclusions and enables specialized templates solely for documentation purposes.
+- **Focused Access Control:** This allows a small team to manage and edit documentation without interference, maintaining high-quality content.
+- **Optimized Data Mining:** Supports easier and more efficient data extraction for feedback and review cycles through Release Notes.
+- **Implementation Reflection:** Mirrors elements from the implementation repositories, providing a high-level knowledge source that is valuable for both business and technical teams.
+- **Release Notes Integration:** Documentation can evolve based on insights from release notes, serving as a dynamic feedback loop back to the documentation repository.
+
+---
+## Living Documentation Regime Features
+
+### Data Mining from GitHub Repositories
+
+This feature allows you to define which repositories should be included in the living documentation process. By specifying repositories, you can focus on the most relevant projects for your documentation needs.
+
+- **Default Behavior**: By default, the action will include all repositories defined in the repositories input parameter. Each repository is defined with its organization name, repository name, and query labels.
+
+### Data Mining from GitHub Projects
+
+This feature allows you to define which projects should be included in the living documentation process. By specifying projects, you can focus on the most relevant projects for your documentation needs.
+
+- **Default Behavior**: By default, the action will include all projects defined in the repositories. This information is provided by the GitHub API.
+- **Non-default Example**: Use available options to customize which projects are included in the documentation.
+  - `project-state-mining: false` deactivates the mining of project state data from GitHub Projects. If set to **false**, project state data will not be included in the generated documentation and project related configuration options will be ignored. 
+  - `projects-title-filter: []` filters the repository attached projects by titles, if list is empty all projects are used.
+      ```json
+        {
+          "organization-name": "absa-group",
+          "repository-name": "living-doc-example-project",
+          "query-labels": ["feature", "bug"],
+          "projects-title-filter": ["Community Outreach Initiatives", "Health Data Analysis"]
+         }
+      ```
+
+### Living Documentation Page Generation
+
+The goal is to provide a straightforward view of all issues in a single table, making it easy to see the overall status and details of issues across repositories.
```

Comment:
Here, I am missing information that the current output implementation is implemented against the mdoc solution. The presence of so many _index.md file can be confusing.

Reply:
I am not sure, if the mdoc is the official IT term or just our naming. See my solution at:

Reply:
More correct can be [mdoc viewer](https://github.com/AbsaOSS/cps-mdoc-viewer).

Reply:
Sentence also updated with a link to the mdoc viewer. See the change at: